db/schema.rb

```diff
@@ -914,6 +914,7 @@
     t.string "preferred_locale"
     t.string "provider"
     t.string "uid"
+    t.string "batch_email_frequency", default: "off"
```

Comment:
Request from @laritakr could we change "off" to be "never". Something about off being a reserved keywords for translations and her UI work.

Reply:
Since you are off today, she will likely make this change herself. Please confirm on your return.